lib/ansible/modules/packaging/language/composer.py

```diff
@@ -52,10 +52,19 @@
         default: null
     working_dir:
         description:
-            - Directory of your project (see --working-dir).
-        required: true
+            - Directory of your project (see --working-dir). This is required when
+              the command is not run globally.
```

Comment:
Probably one thing to add is to document (the obvious) that `working_dir` will be ignored if `global_command=True`

~~~diff
+            - Will be ignored if C(global_command=true).
~~~